src/app/container/container.component.html

```diff
@@ -104,9 +104,14 @@ <h3>
     </div>
   </div>
   <div class="row m-1">
-    <div *ngIf="tool" class="col-sm-12" style="margin-bottom: 5px">
-      <span *ngIf="!labelsEditMode && !starGazersClicked">
-        <mat-chip-list>
+    <div *ngIf="tool">
+      <span *ngIf="(categories$ | async).length > 0" class="inline-block mr-5 mt-1 mb-1">
+        <span class="size-12 mr-2">Categories</span>
+        <app-category-button *ngFor="let category of (categories$ | async)" [category]="category" entryType="tool" class="weight-500"></app-category-button>
```

Comment:
aren't all app-category-button instances visually identical? How come this has a different font-weight?

Reply:
Currently, the app-category-buttons inherit the font-weight, which was a choice I made after misinterpreting the mockups, and probably a mistake.  I am going to harmonize them all to the same weight, and eliminate the weight-500 style.  Will flag you for re-review after I push.